composer.json

```
@@ -18,7 +18,7 @@
 		"codeigniter4/codeigniter4-standard": "^1.0",
 		"fzaninotto/faker": "^1.9@dev",
 		"mikey179/vfsstream": "1.6.*",
-		"phpstan/phpstan": "^0.12.37",
+		"phpstan/phpstan": "^0.12.40",
```

```suggestion
		"phpstan/phpstan": "^0.12",
```
Can we omit the patch part since we're using the caret here?

---

updated.